src/Http/Http/src/BindingAddress.cs

```diff
@@ -179,17 +179,17 @@ public static BindingAddress Parse(string address)
                 }
             }
 
-            pathDelimiterStart = address.IndexOf(":", schemeDelimiterEnd + unixPipeHostPrefixLength, StringComparison.Ordinal);
```

Comment:
Not your fault, but it seems very weird that the char version doesn't (AFAICT) accept a `StringComparison`.  I'm assuming the default is `Ordinal`?

Reply:
I'll double-check this one locally in an IDE.

Reply:
I couldn't find one a `char, int, StringComparison` overload.

The IDE code-fixer also suggests exactly what's here:

![image](https://github.com/user-attachments/assets/43fd8d5f-34a5-4568-b3b8-4d0dfcb9c561)

I'm gonna _guess_ that it does an ordinal check anyway and in this case as it's punctuation it's moot.

Reply:
Sorry, I should have been clearer - there isn't a better overload you could have called.  I don't know why that is, but you don't need to do anything about it.

I just confirmed in the source that the default comparison is `Ordinal` so the change looks correct (if, sadly, less explicit).